src/index.ts

```diff
@@ -106,10 +112,14 @@ export const eventHandler = async (
 
   const params = new URLSearchParams({ ...query, ...payload }).toString()
 
-  const baseURL =
-    eventType === 'remarketing'
-      ? 'https://www.google.com/pagead/1p-user-list'
-      : 'https://www.googleadservices.com/pagead/conversion'
+  let baseURL = ''
+  if (eventType === 'pageview') {
+    return
```

Comment:
this returns out of the entire function - I think a switch statement might be more appropriate for what you're intending

Reply:
The return is on purpose. In case of pageveiew we want the fucntion to stop (so that customer could save GCLID but without reporting anything to GAds).

Reply:
ahh sorry completely missed that core spec detail. Let's return on pageview even earlier then, like line 79?